core/common/src/main/java/alluxio/master/block/BlockId.java

```diff
@@ -14,55 +14,55 @@
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * This class provides a set of methods related to block Ids. Each block Id is a value of long with
+ * This class provides a set of methods related to block IDs. Each block ID is a value of long with
  * the following two parts:
  * <ul>
- * <li>The most significant 5 bytes (40 bits) represent the container Id of this block and</li>
+ * <li>The most significant 5 bytes (40 bits) represent the container ID of this block and</li>
  * <li>The least significant 3 bytes (24 bits) represent the sequence number of this block in the
  * container.</li>
  * </ul>
  */
 @ThreadSafe
-public final class BlockId {
+public final class BlockID {
```

Comment:
sorry but I found that I've already changed this "blockId"  to "blockID" in line 26.